src/views/v2/components/events-bar/search/keyword.php

```diff
@@ -38,4 +38,11 @@ class="tribe-common-form-control-text__input tribe-events-c-search__input tribe-
 		placeholder="<?php echo esc_attr( $placeholder ); ?>"
 		aria-label="<?php echo esc_attr( $aria_label ); ?>"
 	/>
+	<em
+		class="tribe-events-c-search__input-control-icon"
+		aria-label="<?php esc_attr_e( 'Search', 'the-events-calendar' ); ?>"
+		title="<?php esc_attr_e( 'Search', 'the-events-calendar' ); ?>"
+	>
+		<?php $this->template( 'components/icons/search', [ 'classes' => [ 'tribe-events-c-search__input-control-icon-svg' ] ] ); ?>
+	</em>
```

Comment:
we probably don't need to wrap this and the other search icons in an `em`, it's more of a visual flare rather than something really important.